README.md

```diff
@@ -25,16 +25,18 @@ Compilation of all the different mods built to work with Mine Little Pony or tha
 
 ### By other devs
 1. ~[Farmer's Delight](https://modrinth.com/mod/farmers-delight-fabric)~ (only 1.20.1)
```

Comment:
Farmer's Delight Fabric is deprecated. With Unicopia updated you should be able to migrate to the new one with no problem.

https://modrinth.com/mod/farmers-delight-refabricated

Reply:
I actually already migrated just forgot to change the text give me a moment